packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -296,14 +279,32 @@ export const replaceReactComponentsImports = (source: string): boolean => {
 
 /**
  * Removes old fetch import that is no longer used
- * @param source
+ * @param source the src folder
  */
 export const removeFetchImport = (source: string) => {
   const operation = async (filePath: string) => {
+    let hasReplaced = false;
     const fileContent = fs.readFileSync(filePath, "utf8");
-    if (fileContent.match(fetchImportRegex)) {
-      const modifiedContent = fileContent.replace(fetchImportRegex, "");
-      fs.writeFileSync(filePath, modifiedContent, "utf8"); // Update the file content
+    let lines = fileContent.split("\n");
+    for (let i = 0; i < lines.length; i++) {
+      if (
+        lines[i].substring(0, 7).includes("import") &&
```

Comment:
Someone could technically have some commented line somewhere that's like `// fetch fella import guy`

I suppose the substring part helps